util/weekly_build/4_SpackInstall.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+if [ -z $SETUPDONE ]; then . ShellSetup.sh $* ; fi
+
+cd $RUNDIR/${RUNID}
+
+set +x
+. setup.sh
+set -x
+
+INSTALL_OPTS="--show-log-on-error --fail-fast --no-cache $INSTALL_OPTS"
```

Comment:
I would like to have an option to utilize the existing build cache (for faster rebuilds) or ignore it

Reply:
I added an option to not use the --no-cache flag so it'll go with the normal behavior (let me know if the --no-check-signature option is unneeded/unwanted).

Reply:
That will still need the build-cache be added to the spack config, right?

Reply:
Correct (or alternatively point it to $BUILD_CACHE_DIR as a mirror prior to installation)

Reply:
Ok, so for my understanding, are you planning to permanently add the build-cache to the acorn site config and then use it or not with the --no-cache switch? If so, we could retrieve the build-cache directory from the spack config (if we agree on calling it local-binary in consistency with local-source for the source mirror, we could simply push to that as well).

Reply:
Yep that's smart, let's do that :) I'll test that out shortly...

Reply:
Would it make sense to have one directory on each system `.../spack-stack/cache/` and have one mirror? I think Spack will create 'build_cache' and '_source_cache' directories beneath that anyway, so I'm wondering if there's any downside to consolidating and calling them all 'local-cache' or 'spack-stack-cache'.

Reply:
I tried it the consolidated way and like it (just pushed update), but I don't mind separate if you prefer.

Reply:
I think it's ok as long as the location of the consolidated cache for source and binary is configurable and can be different that for the build stage. They may end up on slower nfs filesystems or other types of storage (s3) that are unsuitable for the build stage.